src/translib/transformer/xfmr_bgp.go

```diff
@@ -588,3 +608,174 @@ var DbToYang_bgp_gbl_afi_safi_addr_key_xfmr KeyXfmrDbToYang = func(inParams Xfmr
+    }
+
+    log.Info("In clearall", clear_all)
+    if value, ok := mapData["vrf-name"].(string) ; ok {
+        log.Info("In vrf", value)
+        if value != "" {
+            vrf_name = "vrf " + value + " "
+        }
+    }
+
+    if value, ok := mapData["family"].(string) ; ok {
+        if value == "IPv4" {
+            af_str = "ipv4 "
+        } else if value == "IPv6" {
+            af_str = "ipv6 "
+        }
+    }
+
+    if value, ok := mapData["all"].(bool) ; ok {
+        if value == true {
+           all = "* "
+        }
+    }
+
+    if value, ok := mapData["external"].(bool) ; ok {
+        if value == true {
+           external = "external "
+        }
+    }
+
+    if value, ok := mapData["address"].(string) ; ok {
+        if value != "" {
+            neigh_address = value + " "
+        }
+    }
+
+    if value, ok := mapData["interface"].(string) ; ok {
+        if value != "" {
+            intf = value + " "
+        }
+    }
+
+    if value, ok := mapData["asn"].(float64) ; ok {
+        _asn := uint64(value)
+        asn = strconv.FormatUint(_asn, 10)
+        asn = asn + " "
+    }
+
+    if value, ok := mapData["prefix"].(string) ; ok {
+        if value != "" {
+            prefix = "prefix " + value + " "
+        }
+    }
+
+    if value, ok := mapData["peer-group"].(string) ; ok {
+        if value != "" {
+            peer_group = "peer_group " + value + " "
+        }
+    }
+
+    if value, ok := mapData["in"].(bool) ; ok {
+        if value == true {
+           in = "in "
+        }
+    }
+
+    if value, ok := mapData["out"].(bool) ; ok {
+        if value == true {
+            out = "out "
+        }
+    }
+
+    if value, ok := mapData["soft"].(bool) ; ok {
+        if value == true {
+            soft = "soft "
+        }
+    }
+
+    log.Info("In rpc_clear_bgp", clear_all, vrf_name, af_str, all, neigh_address, intf, asn, prefix, peer_group, in, out, soft)
+
+    if clear_all != "" {
+        cmd = "clear ip bgp " + clear_all
+    } else {
+        cmd = "clear ip bgp "
+        if vrf_name != "" {
+            cmd = "clear ip bgp " + vrf_name
+        }
+
+        if af_str != "" {
+            cmd = cmd + af_str
+        }
+
+        if neigh_address != "" {
+            cmd = cmd + neigh_address
+        }
+
+        if intf != "" {
+            cmd = cmd + intf
+        }
+
+        if prefix != "" {
+            cmd = cmd + prefix
+        }
+
+        if peer_group != "" {
+            cmd = cmd + peer_group
+        }
+
+        if external != "" {
+            cmd = cmd + external
+        }
+
+        if asn != "" {
+            cmd = cmd + asn
+        }
+
+        if all != "" {
+            cmd = cmd + all
+        }
+
+        if soft != "" {
+            cmd = cmd + soft
+        }
+
+        if in != "" {
+            cmd = cmd + in
+        }
+
+        if out != "" {
+            cmd = cmd + out
+        }
+
+    }
+    cmd = strings.TrimSuffix(cmd, " ")
+    exec_vtysh_cmd (cmd)
```

Comment:
can you handle the error and fill status accordingly.

Reply:
clear is not returning proper json for FRR